setup.rb

```diff
@@ -97,12 +100,14 @@
   :devel_user => options[:devel_user],
   :deployment_dir => options[:deployment_dir]
 )
+
 installer = KatelloDeploy::Installer.new(
   :installer_options => installer_options,
   :skip_installer => options[:skip_installer],
-  :type => options[:install_type],
-  :local_path => (File.directory?('./katello-installer') && options[:version] == 'nightly') ? './katello-installer' : nil
+  :scenario => options[:scenario],
+  :root_dir => Dir.pwd
 )
+`gem install kafo`
```

Comment:
The right Kafo is in nightlies already, gem installation produces gem version warnings